posthog/hogql/property.py

```diff
@@ -129,7 +129,7 @@ def property_to_expr(
                 return ast.Or(exprs=exprs)
 
         chain = ["person", "properties"] if property.type == "person" and scope != "person" else ["properties"]
-        field = ast.Field(chain=chain + [property.key])
+        field = ast.Field(chain=chain + property.key.split("."))
```

Comment:
Drawing attention to this, theoretically could be breaking as it unescapes any `.`.

i.e. `a.b` was previously `"properties"."a.b."` and now will be `"properties"."a"."b"`

Reply:
Hmm, sorry, got to block this, as it's a global change in the `hogql` folder. We have users who have actual dots in their property name strings, and this will break some filter somewhere for them. The event/feature/property filter is made to work with the current taxonomic filter, which presents just the first layer of properties and has never supported more.

You can use the `hogql` template helper like ```
hogql`SELECT properties.${hogql.identifier('odd property')} FROM events`
```
if you want to cleanly escape multiple properties into a hogql string.

Reply:
This is fixed by creating hogql filters on the front end